discord/interactions/deleteConfigCommand.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  ActionRowBuilder,
+  ButtonBuilder,
+  ButtonInteraction,
+  ButtonStyle,
+  ChatInputCommandInteraction,
+  Client,
+  ModalActionRowComponentBuilder,
+  ModalBuilder,
+  ModalSubmitInteraction,
+  REST,
+  SelectMenuBuilder,
+  SelectMenuComponentOptionData,
+  SelectMenuInteraction,
+  TextInputBuilder,
+  TextInputStyle,
+} from "discord.js";
+import { assertAdmin } from "./permissions";
+
+import { DiscordServerConfig } from "../../db/entity/DiscordServerConfig";
+import {
+  DiscordServerConfigRepository,
+  DiscordMemberRepository,
+} from "../../db";
+import { getRoles, isBotRole } from "../role";
+import starkyModules from "../../starkyModules";
+import config from "../../config";
+
+export const handleDeleteConfigCommand = async (
+  interaction: ChatInputCommandInteraction,
+  client: Client,
+  restClient: REST
+) => {
+  await assertAdmin(interaction);
+  if (!interaction.guildId) return;
+
+  const configurations = await DiscordServerConfigRepository.findBy({
+    discordServerId: interaction.guildId,
+  });
+  // Showing configurations
+  const options = configurations.map((config) => ({
+    label: config.id.toString(),
```

Comment:
can you generate a clean label? Because the id actually doesn't help the user
The best way to do it is probably to add a method on the starky module:

```
const configName = (
  starknetNetwork: "mainnet" | "goerli",
  starkyModuleConfig: StarkyModuleConfig
): string
```

For ERC721 module for instance it would return

`Goerli - ERC-721 - <0xaddress of ERC721 contract>`

That would make this dropdown really useful!